elasticdl/python/data/reader/data_reader.py

```diff
@@ -1,11 +1,47 @@
 from abc import ABC, abstractmethod
 
+from elasticdl.python.common.dtypes import MAXCOMPUTE_DTYPE_TO_TF
+
 
 class Metadata(object):
+    """ Metadata of a dataset containing column name and dtype
+
+    Attributes:
+        column_names: A list with column names
+        column_dtypes: A dict where the key is a column name
+            and the value is a dtype
+    """
+
     def __init__(self, column_names, column_dtypes=None):
         self.column_names = column_names
         self.column_dtypes = column_dtypes
```

Comment:
When do we use `column_names`? And following code is FYI.

```python
@property
def column_dtypes(self):
    return self.column_dtypes

@column_dtypes.setter
def column_dtypes(self, col_dtypes):
    self.column_dtypes = col_dtypes
```

Reply:
We use `column_names` to create a dataset  in `dataset_fn`. The features of the datset is a dict like {column_name: tensor}.